src/index.js

```diff
@@ -7,6 +7,8 @@ import { SourceMapConsumer } from 'source-map';
 import { SourceMapSource, RawSource, ConcatSource } from 'webpack-sources';
 import RequestShortener from 'webpack/lib/RequestShortener';
 import ModuleFilenameHelpers from 'webpack/lib/ModuleFilenameHelpers';
+import validateOptions from 'schema-utils';
+import schema from '../options.json';
```

Comment:
yea, I thought that you wanted me to include that with the --copy-files change.. ill switch it back